framework/src/bundle/BundlePrivate.cpp

```diff
@@ -65,14 +65,14 @@ void BundlePrivate::Stop(uint32_t options)
 
   {
     auto l = this->Lock();
+    US_UNUSED(l);
 
     // 1:
     if (state == Bundle::STATE_UNINSTALLED) {
       throw std::logic_error("Bundle is uninstalled");
     }
 
     // 2: If an operation is in progress, wait a little
-    // Note: This step hass been removed with removal of BundleThread
 
     // 3:
```

Comment:
you still want to remove this comment then re-number the remainder of the comments.